pandas/core/frame.py

```diff
@@ -4049,15 +4063,17 @@ def _sanitize_column(self, value):
         return value
 
     @property
-    def _series(self):
+    def _series(self) -> Dict[IndexLabel, Series]:
```

Comment:
IndexLabel is Union[Hashable, Sequence[Hashable]]

the dictionary keys will be the items from Index.__iter__

so a tuple for a MultiIndex, which is a Sequence but we should be more precise for return types

is the return type of Index.__iter__ typed? If so should be the same. or sometimes better to type lower level functions first.

this module is difficult to type since it is high level and user facing, probably should be one of the last modules to type

Reply:
okay will remove for now